app/src/main/java/com/lagradost/quicknovel/providers/HiraethTranslationProvider.kt

```diff
@@ -0,0 +1,123 @@
+package com.lagradost.quicknovel.providers
+
+import com.lagradost.quicknovel.*
+import com.lagradost.quicknovel.MainActivity.Companion.app
+import org.jsoup.Jsoup
+
+import android.util.Log
+
+
+open class HiraethTranslationProvider : MainAPI() {
+    override val name = "HiraethTranslation"
+    override val mainUrl = "https://hiraethtranslation.com"
+    override val hasMainPage = true
+    override val iconId = R.drawable.icon_hiraethtranslation
+    override val iconBackgroundId = R.color.hiraethtranslation_header_color
+
+    override suspend fun loadMainPage(
+        page: Int,
+        mainCategory: String?,
+        orderBy: String?,
+        tag: String?
+    ): HeadMainPageResponse {
+        val url = "$mainUrl/page/$page/?s&post_type=wp-manga&m_orderby=latest"
```

Comment:
Hardcoded `m_orderby=latest` and `post_type=wp-manga` ? 

You can override orderBys and mainCategories to make it possible to sort new items